api/src/test/java/dk/dbc/search/work/presentation/api/jpa/RecordEntityIT.java

```diff
@@ -59,7 +58,6 @@ public void testSaveLoad() throws Exception {
         wi.creators = Arrays.asList("hans andersen");
         wi.subjects = Collections.singleton("emne");
```

Comment:
Spørgsmål: hvorfor to forskellige måder at lave en singleton liste?

Reply:
Den ene laver en liste, den anden et set